docs/source/examples.rst

```diff
@@ -53,6 +53,38 @@ An analysis on a single shot
 	# shots in a multishot analysis:
 	run.save_result('mot loadrate', c)
 
+Single shot analysis with global file opening
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+	from lyse import *
+
+	# Instantiate Run object and open
+	# Globally opening the shot keeps the h5 file open
+	# This prevents excessive opening and closing of the file
+	# which can slow down the analysis
+	with Run(path).open('r+') as shot:
+
+		# Obtaining a trace:
+		t, mot_fluorecence = run.get_trace('mot fluorecence')
```

Comment:
Needs to be `shot.`

Reply:
This is what I get for typing and not checking...